templates/cloud/cloud-computing.html

```diff
@@ -0,0 +1,494 @@
+{% extends "cloud/_base-cloud.html" %}
+
+{% block title %}What is a cloud computing?{% endblock %}
+
+{% block meta_description %}Cloud computing is an on-demand delivery of IT resources over a network. The cloud resource provisioning process is fully automated, leading to increased agility and significant cost savings.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1oIggNSXR5sfl0Ab35x2NIL7I858I7QdPCUXeQI8IIPc/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-7">
+      <h1>What is cloud computing?</h1>
+      <p>Cloud computing is the on-demand delivery of IT resources over a network. In traditional data centres, compute and storage resources used to be allocated manually by a dedicated IT team. In the cloud, this process is fully automated, leading to increased agility and significant cost savings.</p>
+      <p>Cloud computing is a de facto standard for modern data centre implementation and a foundation for <a href="/blog/what-is-cloud-native">cloud-native application</a> deployments. It is used by the majority of organisations worldwide. There is no reason why yours shouldn’t be next.</p>
```

Comment:
```suggestion
      <p>Cloud computing is a <em>de facto</em> standard for modern data centre implementation and a foundation for <a href="/blog/what-is-cloud-native">cloud-native application</a> deployments. It is used by the majority of organisations worldwide. There is no reason why yours shouldn’t be next.</p>
```
In the doc this is in _italics_

Reply:
Oh wow I didn't catch that, thanks.